<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-P81C98J" name="RetryMultiple" tests="22" failures="2" timestamp="29 May 2022 20:57:19 GMT" time="0.5" errors="0">
  <testcase name="Test1" time="0.031" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.016" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.016" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.0" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.016" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.0" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.0" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.016" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.0" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.016" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.0" classname="WebPgeScripts.check$Test001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at WebPgeScripts.check$Test001.Test1(check.java:14)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase name="Test2" time="0.0" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.015" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.016" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.0" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.062" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.016" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.0" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.0" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.0" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.016" classname="WebPgeScripts.check$Test001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.0" classname="WebPgeScripts.check$Test001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at WebPgeScripts.check$Test001.Test2(check.java:20)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
</testsuite> <!-- RetryMultiple -->
